test/testhelpers/helpers.go

```diff
@@ -66,3 +67,66 @@ func NewPrestoTable(name, namespace string, columns []hive.Column) *v1alpha1.Pre
 		},
 	}
 }
+
+type ReportDataSourceStore struct {
+	datasources map[string]*v1alpha1.ReportDataSource
+}
+
+func NewReportDataSourceStore(datasources []*v1alpha1.ReportDataSource) (store *ReportDataSourceStore) {
+	m := make(map[string]*v1alpha1.ReportDataSource)
+	for _, dataSource := range datasources {
+		m[dataSource.Namespace+"/"+dataSource.Name] = dataSource
+	}
+	return &ReportDataSourceStore{m}
+}
+
+func (store *ReportDataSourceStore) GetReportDataSource(namespace, name string) (*v1alpha1.ReportDataSource, error) {
+	dataSource, ok := store.datasources[namespace+"/"+name]
+	if ok {
+		return dataSource, nil
+	} else {
```

Comment:
Can drop this `else` clause, and the other ones on line #'s 108 & 129

Reply:
Good point.